admin_manual/configuration/server/security/password_policy.rst

```diff
@@ -0,0 +1,61 @@
+===============
+Password Policy
+===============
+
+.. _password_policy_label:
+
+.. figure:: ../../../images/configuration/server/security/password-policy-app.png
+   :alt: The Password Policy application
+
+From the 2.0.0 release of `the Password Policy app`_, ownCloud administrators (both enterprise **and** community edition) have the option of installing and enabling the application.
+The Password Policy application enables administrators to define password requirements for user passwords and public links.
+
+**User Passwords**
+
+- Specify valid password requirements.
+- Specify a password expiration period.
+- Specify expiration dates for public link shares.
+- Forced password change on first login.
+- Disallowing passwords that match a configurable number of previous passwords (defaults to the previous 3).
+- Users can be notified a configurable number of days before their password expires.
+- Users will be notified when their password has expired.
+
+**Public links**
+
```

Comment:
Hmm, seems the config UI is really misleading here.
Let me sort this again:
![password-policy-app](https://user-images.githubusercontent.com/21193500/43003234-181b1ee8-8c2c-11e8-8ba7-dc195833e6d7.png)

There are three sections:
1) Minimum password requirements for user accounts and public links
=> All of them apply to user passwords and public link passwords **except** the first one, "X different than last passwords" ("password history"). That one only applies to user passwords. Just realizing that the UI really is a nightmare :(

2) User password policies
=> These are the features that only apply to user passwords: Password expiration, the reminder notification for expiration and forcing password change on first login.
Actually, this is where the password history part belongs to. Raised a ticket to improve this https://github.com/owncloud/password_policy/issues/72.

3) Public link expiration policies
=> Enforce different expiration dates of public links depending on whether a password has been set or not. :heavy_check_mark: